_posts/2022-04-13-cockpit-267.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cockpit 267
+author: pitti
+date: '2022-04-13'
+tags: cockpit, machines
+slug: cockpit-267
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 267 and cockpit-machines 266:
+
+
+## FIPS crypto policy support
+
+The Federal Information Processing Standard (FIPS) is a computer security standard developed by the U.S. Government and industry working group to validate the quality of cryptographic modules. See the official FIPS publications at [NIST Computer Security Resource Center](https://csrc.nist.gov/publications/fips).
+
+The crypto policy dialog can now enable or disable FIPS mode, using the [fips-mode-setup command](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening). It will also validate that the kernel and user-space configuration are consistent, and allow you to fix them when necessary.
+
+![screenshot of fips crypto policy support]({{ site.baseurl }}/images/267-fips-crypto-policy-support.png)
+
+## Machines: Redesign VM creation
+
+The content of the Virtual machine creation dialog has been split into a mandatory Details configuration and an optional Automation configuration.
+
+Clicking on "Create and run" will create, immediately start, and install the VM. Clicking on "Create and edit" will create the VM in a shut-off state and redirect the user to the details page, where further configuration of the VM may be done before the installation is started.
```

Comment:
```suggestion
Clicking on "Create and run" will create, immediately start, and install the VM. Clicking on "Create and edit" will create the VM in a "shut-off" state and redirect the user to the details page, where the VM may be further customized before starting installation.
```

The sentence is rather wordy, so I dropped a few words and reorganized it a little.